README-gradle.md

```diff
@@ -0,0 +1,191 @@
+         apply plugin: 'checkstyle'
+         /* add apply plugin: "testrunner" to the end of the subproject{} */
+         apply plugin: 'testrunner'
+       }
+         
+       /* add apply plugin: "testrunner" to the root */ 
+       apply plugin: 'testrunner'
+       ```
+   * If only `allprojects{}`method is in gradle.build, the command is added to  `allprojects{}` only.
+     * Example: 
+
+       ```groovy
+       buildscript {
+          repositories {
+             mavenCentral()
+          }
+       
+          dependencies {
+             classpath group: 'edu.illinois.cs',
+             name: 'idflakies', 
+             version: '1.1.0'
+       
+             classpath group: 'edu.illinois.cs',
+             name: 'testrunner-gradle-plugin', 
+             version: '1.2'
+          }
+       }
+       
+       allprojects {
+         repositories {
+         	mavenCentral()
+         }
+         /* add apply plugin: "testrunner" to the end of the allprojects{} only */
+         apply plugin: 'testrunner'
+       }
+       ```
+   * If only `subprojects{}` method is in gradle.build, the command is added to both the end of `subprojects{}` and the root. 
+
+     * Example: 
+
+       ```groovy
+       buildscript {
+          repositories {
+             mavenCentral()
+          }
+       
+          dependencies {
+             classpath group: 'edu.illinois.cs',
+             name: 'idflakies', 
+             version: '1.1.0'
+       
+             classpath group: 'edu.illinois.cs',
+             name: 'testrunner-gradle-plugin', 
+             version: '1.2'
+          }
+       }
+       
+       subprojects {
+         apply plugin: 'maven'
+         apply plugin: 'signing'
+         apply plugin: 'checkstyle'
+         /* add apply plugin: "testrunner" to the end of the subproject{} */
+         apply plugin: 'testrunner'
+       }
+       
+       /* add apply plugin: "testrunner" to the root */ 
+       apply plugin: 'testrunner'
+       ```
+   * If there's no `allprojects{}` and `subprojects{}` in gradle.build, only add the command to the root.
+     * Example: 
+
+       ```groovy
+       buildscript {
+          repositories {
+             mavenCentral()
+          }
+       
+          dependencies {
+             classpath group: 'edu.illinois.cs',
+             name: 'idflakies', 
+             version: '1.1.0'
+       
+             classpath group: 'edu.illinois.cs',
+             name: 'testrunner-gradle-plugin', 
+             version: '1.2'
+          }
+       }
+       
+       apply plugin: 'testrunner'
+       ```
+3. For each subproject with build.gradle file in its directory, append `apply plugin: 'testrunner'` to the end of the build.gradle in each subproject. 
+### Use locally built version of iDFlakies 
+The use of locally built version of iDFlakies needs the user to modify the dependency 
+Then go to iDFlakies root directory, and run: 
+```
+mvn clean install
+```
+```groovy
```

Comment:
Please add some text before this code snippet. I'm assuming the point here is that if one chooses to use locally built version of iDFlakies they would have to modify the build.gradle similar to what is shown in this code snippet right? 

You may also consider adding the command to automatically use the ```gradle_modify/modify_gradle.sh``` script